### PR TITLE
refactor: [M3-8649] - Move `Radio` to ` at linode/ui` package

### PR DESCRIPTION
**Pull request by pmakode-akamai**
## Description 📝
Moves the `Radio` component, story and unit tests to the `@linode/ui` package and updates existing imports.

## Changes  🔄
- Move Radio component to @linode/ui package
- Move needed icons to ui/src/assets/icons directory
- Update existing imports
- Add `vite-plugin-svgr` plugin to `@linode/ui` package
- Remove typecast from plugins configuration

## Target release date 🗓️
N/A

## How to test 🧪
- Confirm all checks pass
- Confirm nothing on CM breaks

## As an Author I have considered 🤔

*Check all that apply*

- [ ] 👀 Doing a self review
- [x] ❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
- [x] 🤏 Splitting feature into small PRs
- [x] ➕ Adding a [changeset](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md#writing-a-changeset)
- [ ] 🧪 Providing/Improving test coverage
- [ ] 🔐 Removing all sensitive information from the code and PR description
- [ ] 🚩 Using a feature flag to protect the release
- [ ] 👣 Providing comprehensive reproduction steps
- [ ] 📑 Providing or updating our documentation
- [ ] 🕛 Scheduling a pair reviewing session
- [ ] 📱 Providing mobile support
- [ ] ♿  Providing accessibility support

### CODE REVIEW

File: packages/ui/src/components/Radio/Radio.stories.tsx [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import { Box } from '@linode/ui';
 import React from 'react';
 
-import { FormControlLabel } from '../FormControlLabel';
-import { RadioGroup } from '../RadioGroup';
+import { FormControlLabel } from '../../../../manager/src/components/FormControlLabel';
+import { RadioGroup } from '../../../../manager/src/components/RadioGroup';
```
Comment (coliu-akamai) [inline packages/ui/src/components/Radio/Radio.stories.tsx:5]: just a heads up, we're trying to [avoid having `linode/ui` be dependent on `manager`](https://github.com/linode/manager/pull/11125#discussion_r1813519608) -- we'll need to move these two components over first

Comment (pmakode-akamai) [reply packages/ui/src/components/Radio/Radio.stories.tsx:5]: I agree. I'll update these imports once the `FormControlLabel` and `RadioGroup` components are moved to the `ui` package

File: packages/ui/vitest.config.ts [hunk 2]
```diff
@@ -1,6 +1,8 @@
-import { defineConfig } from 'vitest/config';
+import { defineConfig, UserConfig } from 'vitest/config';
+import svgr from 'vite-plugin-svgr';
 
 export default defineConfig({
+  plugins: [svgr({ exportAsDefault: true })] as UserConfig['plugins'],
```
Comment (hkhalil-akamai) [inline packages/ui/vitest.config.ts:5]: I guess this typecast isn't needed anymore as removing it has no negative effects. Maybe we can remove it from other places as well.

```suggestion
  plugins: [svgr({ exportAsDefault: true })],
```

File: packages/manager/cypress/vite.config.ts [hunk 3]
```diff
@@ -3,14 +3,12 @@ import { URL } from 'url';
...
 
-import type { UserConfig } from 'vite';
-
 // ESM-friendly alternative to `__dirname`.
 const DIRNAME = new URL('.', import.meta.url).pathname;
 
 export default defineConfig({
   // @todo Remove this `as` when we upgrade our package manager. Yarn v1's hoisting behavior is causing a type error
-  plugins: [react(), svgr({ exportAsDefault: true })] as UserConfig['plugins'],
+  plugins: [react(), svgr({ exportAsDefault: true })],
```
Comment (coliu-akamai) [inline packages/manager/cypress/vite.config.ts:11]: we can remove the @todo comment here (and for the other vite.config.ts)

I am a bit curious what that type error was/why it seems to no longer be appearing even though we haven't yet changed our package manger 👀

**Review by coliu-akamai — APPROVED** [review]
thanks @pmakode-akamai!

**Review by harsh-akamai — APPROVED** [review]
Code Review ✅ 
Changes in Cloud ✅ 

Thanks @pmakode-akamai for this and the PRs blocking to this component! 🎉

**Pull request merged**